### PR TITLE
Experiment with unique test names

### PR DESCRIPTION
**Pull request by roji**

### CODE REVIEW

File: test/Npgsql.Tests/Replication/TestDecodingReplicationTests.cs [hunk 1]
```diff
@@ -94,14 +96,23 @@ public Task UpdateForDefaultReplicaIdentity()
...
+                        Assert.That(message.Data, Does.StartWith("BEGIN "));
 
-                    // Update
-                    message = await NextMessage(messages);
-                    Assert.That(message.Data,
-                        Is.EqualTo($"table public.{tableName}: UPDATE: id[integer]:1 name[text]:'val1'"));
+                        // Update
+                        message = await NextMessage(messages);
+                        if (message.Data.StartsWith("COMMIT "))
+                            continue;
```
Comment (Brar) [inline test/Npgsql.Tests/Replication/TestDecodingReplicationTests.cs:110]: I understand that this fixes a failing test but isn't that jus covering up a problem that shouldn't happen and that we don't fully understand?

Comment (Brar) [reply test/Npgsql.Tests/Replication/TestDecodingReplicationTests.cs:110]: BTW: test_decoding has the `SkipEmptyXacts` that should to the same within the logical decoding plugin.

Comment (roji) [reply test/Npgsql.Tests/Replication/TestDecodingReplicationTests.cs:110]: https://www.postgresql.org/message-id/E1XOSIv-0003rO-9d@gemulon.postgresql.org

**Review by Brar — APPROVED** [review]
LGTM

**Comment by roji** [comment]
Replaced by #13

**Pull request closed without merging**